draft-packages/illustration/KaizenDraft/Illustration/Players/VideoPlayer.tsx

```diff
@@ -101,15 +101,25 @@ export const VideoPlayer = ({
     if (prefersReducedMotion) {
       videoElement.pause()
     } else if (autoplay && !prefersReducedMotion) {
-      videoElement.play().catch(e => {
```

Comment:
```suggestion
videoElement.play()?.catch(e => {
```

Would optional chaining possibly help? Although of course this means that if `catch` doesn't exist, we get no error logs. How much do we care about them?

Reply:
I always forget about optional chaining. We don't care about the error as it fallsback to not playing the animation. Will update.

Reply:
I ended up typing it so that it's explicit to people why we have optional chaining there. The `dom.ts` that we reference doesn't show that it could be undefined.

Reply:
I don't think you need the outer try/catch anymore with the optional chaining.

I'm not a big fan of type-casting though, because it opens us up to missing errors if there are any changes in the future. Certainly worth documenting though, but I think a comment is enough in this situation.

Reply:
The `.play` can fail as well so you'd still need the outer try catch

Reply:
removed the typecasting and left the comment

Reply:
Ah okay. I'll trust you tested it then :)